README.md

```diff
@@ -1,4 +1,6 @@
-# mirsg.infrastructure Ansible Collection
+# mirs
+
+.infrastructure Ansible Collection
```

Comment:
```suggestion
# mirsg.infrastructure Ansible Collection
```

Reply:
Whoops, not sure what happened there, thanks for spotting. This has been merged so I've created a new PR https://github.com/UCL-MIRSG/ansible-collection-infra/pull/29